lambda_src/response_storage.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Dict, Text
+
+import boto3
+
+AWS_REGION = os.environ[
+    'AWS_REGION'
+]
+DYNAMODB_TABLE = 'geff-requests'
+DYNAMODB_RESOURCE = boto3.resource('dynamodb', region_name=AWS_REGION)
+table = DYNAMODB_RESOURCE.Table(DYNAMODB_TABLE)
```

Comment:
how will this table be created?

Reply:
Have added resources for it [here](https://github.com/Snowflake-Labs/terraform-snowflake-api-integration-with-geff-aws/pull/71/files)